CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,37 @@
 
 import Foundation
 
-print("Hello, World!")
+//아이스크림 부분을 그리는 함수
+/*
+func drawIceCreamPart(){
+    for _ in 1...8{
+        print("***********")
+    }
+}
+ */
 
+drawIceCreamPart()
+
+//이렇게 하면 줄바꿈을 두번씩 함. -> 한번씩만 할 수 있는 방법은 없을까?
+
+func drawIceCreamPart(){
+    for _ in 1...8{
+        for _ in 1...11{
+            print ("*", terminator: "")
```

Comment:
함수를 호출할 때 [API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에 따르면 괄호는 띄어쓰지 않고 호출합니다! 이부분 역시 문서를 잙 읽어보시면 좋을거 같습니다!
```suggestion
            print("*", terminator: "")
```